bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/StepArgument/HumanReadableIntegerExpression.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Capgemini.PowerApps.SpecFlowBindings.Steps.StepArgument
```

Comment:
I don't think this additional class is needed. It'd be easier to just return an `int` from the transformation

Reply:
deleted